Readme.md

```diff
@@ -0,0 +1,124 @@
+# Yoti Joomla SDK #
+
+Welcome to the Yoti Joomla SDK. This repo contains the tools you need to quickly integrate your Joomla back-end with Yoti, so that your users can share their identity details with your application in a secure and trusted way.    
+
+## Table of Contents
+
+1) [An Architectural view](#an-architectural-view) -
+High level overview of integration
+
+2) [References](#references)-
+Guides before you start
+
+3) [Installing the SDK](#installing-the-sdk)-
+How to install our SDK
+
+4) [Plugin Setup](#plugin-setup)-
+How to set up the plugin in Drupal
+
+5) [Linking existing accounts to use Yoti authentication](#linking-existing-accounts-to-use-yoti-authentication)
+
+6) [API Coverage](#api-coverage)-
+Attributes defined
+
+7) [Support](#support)-
+Please feel free to reach out
+
+## An Architectural view
+
+Before you start your integration, here is a bit of background on how the integration works. To integrate your application with Yoti, your back-end must expose a GET endpoint that Yoti will use to forward tokens.
+The endpoint can be configured in the Yoti Dashboard when you create/update your application. For more information on how to create an application please check our [developer page](https://www.yoti.com/developers/documentation/#login-button-setup).
+
+The image below shows how your application back-end and Yoti integrate into the context of a Login flow.
+Yoti SDK carries out for you steps 6, 7 and the profile decryption in step 8.
+
+![alt text](https://github.com/getyoti/yoti-joomla/raw/master/login_flow.png "Login flow")
+
+
+Yoti also allows you to enable user details verification from your mobile app by means of the Android (TBA) and iOS (TBA) SDKs. In that scenario, your Yoti-enabled mobile app is playing both the role of the browser and the Yoti app. Your back-end doesn't need to handle these cases in a significantly different way. You might just decide to handle the `User-Agent` header in order to provide different responses for desktop and mobile clients.
+
+## References
+
+* [AES-256 symmetric encryption][]
+* [RSA pkcs asymmetric encryption][]
+* [Protocol buffers][]
+* [Base64 data][]
+
+[AES-256 symmetric encryption]:   https://en.wikipedia.org/wiki/Advanced_Encryption_Standard
+[RSA pkcs asymmetric encryption]: https://en.wikipedia.org/wiki/RSA_(cryptosystem)
+[Protocol buffers]:               https://en.wikipedia.org/wiki/Protocol_Buffers
+[Base64 data]:                    https://en.wikipedia.org/wiki/Base64
+
+## Installing the SDK
+
+To import the Yoti SDK inside your project:
+
+1) Log on to the admin console of your Joomla website. e.g. Https://www.joomladev.com/administrator
+2) Navigate to at `Extensions-> Manage -> Install` and do one of the following:
+- click `Install from Web` link and Search for Yoti
+- click `Upload Package File` then you can download the package from [here](https://extensions.joomla.org/yoti).
```

Comment:
click `Upload Package File` and upload the downloaded zip file from [our extension page]